elasticsearch/Dockerfile.centos7

```diff
@@ -15,16 +15,16 @@ ENV ES_CLOUD_K8S_VER=2.4.4_01 \
     INSTANCE_RAM=512G \
     JAVA_VER=1.8.0 \
     NODE_QUORUM=1 \
-    OSE_ES_VER=2.4.4.13 \
-    PROMETHEUS_EXPORTER_VER=2.4.4.0 \
+    OSE_ES_VER=2.4.4.22 \
```

Comment:
is this what we have in the es5.x branch?

Reply:
I do second @ewolinetz this is looking strange.